types/tinycon/index.d.ts

```diff
@@ -3,15 +3,15 @@
 // Definitions by: Daniel Waxweiler <https://github.com/dwaxweiler>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-export function setBubble(count: number): void;
+export function setBubble(count: number|string|null): void;
```

Comment:
Please add spaces between the types and bars to be consistent.

Reply:
Done.